stacks-signer/src/config.rs

```diff
@@ -38,6 +36,8 @@ use wsts::state_machine::PublicKeys;
 pub type SignerKeyIds = HashMap<u32, Vec<u32>>;
 
 const EVENT_TIMEOUT_MS: u64 = 5000;
+//TODO: make this zero once special cased transactions are allowed in the stacks node
+const TX_FEE_MS: u64 = 10_000;
```

Comment:
I think we should make a separate ticket to add the fee estimation endpoint to `stacks_client`. 10,000 microstacks would be too low a lot of the time.

Reply:
Furthermore, votes for aggregate public keys can be zero-fee transactions

Reply:
No we still need this. For pre epoch 3. we need to pass in a fee for cast aggregate public key vote.